### DISCUSSION TITLE
change default interactive option

### DISCUSSION THREAD

**Discussion by matmaer** [twpayne/chezmoi#4138] [Ideas]
Make chezmoi safer by allowing the diffs to show up by default for local apply/add/re-add. My workaround is a fish abbreviation that adds "--interactive" to the chezmoi command as a safety net.

You could let settings like "[git] autoCommit = true", override. the idea is useful only in a local interactive context I think.

Happy chezmoi camper here by the way. Just using the idea discussion category. Cheers and nothing but kudos for the project.

**Comment by twpayne** [comment]
OK, #4139 adds an `interactive` config variable so you can set `interactive = true` in your config file. Be aware that you'll need to update any scripts that rely on chezmoi being non-interactive to explicitly pass the `--interactive=false` flag to override this.

**Reply by matmaer** [reply]
Wow man thanks so much, this is perfect!
